### PR TITLE
Test master 1

### PR DESCRIPTION
**Pull request by ue71603**

### CODE REVIEW

File: gtfs-netex-test/any-to-epip.py [hunk 1]
```diff
@@ -37,31 +68,47 @@ def conversion(input_filename: str, output_filename: str):
...
         time_demand_type: TimeDemandType = parser.parse(element, TimeDemandType)
         time_demand_types.append(time_demand_type)
 
+    for element in base_tree.iterfind(".//{http://www.netex.org.uk/netex}TimeDemandType"):
+        time_demand_type: TimeDemandType = parser.parse(element, TimeDemandType)
+        time_demand_types.append(time_demand_type)
 
-
-    codespace = Codespace(id="OPENOV", xmlns="OPENOV", xmlns_url="http://openov.nl/")
+    codespace = Codespace(id="OPENOV", xmlns="OPENOV", xmlns_url="http://openov.nl/") TODO take from file
```
Comment (skinkie) [inline gtfs-netex-test/any-to-epip.py:111]: You want to have the DefaultCodespace or something?

File: gtfs-netex-test/any-to-epip.py [hunk 2]
```diff
@@ -89,7 +136,11 @@ def conversion(input_filename: str, output_filename: str):
     timetabledpassingtimesprofile.getTimetabledPassingTimes(clean=True)
 
     service_frame: ServiceFrame
-    service_frame = parser.parse(tree.find(".//{http://www.netex.org.uk/netex}ServiceFrame"), ServiceFrame)
+    if (tree.find(".//{http://www.netex.org.uk/netex}ServiceFrame") != None):
```
Comment (skinkie) [inline gtfs-netex-test/any-to-epip.py:139]:
```suggestion
    if (tree.find(".//{http://www.netex.org.uk/netex}ServiceFrame") is not None):
```

File: gtfs-netex-test/GtfsNeTEx.py [hunk 3]
```diff
@@ -61,34 +61,62 @@ def getCodespaceAndDataSource(self) -> (Codespace, DataSource, Version, VersionF
         with self.conn.cursor() as cur:
             cur.execute(feed_info_sql)
             df = cur.df()
+            # load dummy feed_info
+            print("feed_info.txt missing. Loading dummy values.")
+            short_name=self.getShortName("UNKNOWN")
+            codespace=Codespace(id="{}:Codespace:{}".format(short_name, short_name), xmlns=short_name,
+                                  xmlns_url="https://unknown.net", description="Unknown")
```
Comment (skinkie) [inline gtfs-netex-test/GtfsNeTEx.py:68]:
```suggestion
            codespace=Codespace(id=f"{short_name}:Codespace:{short_name}", xmlns=short_name,
                                  xmlns_url="https://unknown.net", description="Unknown")
```

**Comment by ue71603** [comment]
no longer relevant

**Pull request closed without merging**